File: my-folder/0217-contains-duplicate/solution.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet uniqueElements = new HashSet();
        for (int i = 0; i < nums.length; ++i)
        {
            if (uniqueElements.contains(nums[i]))
                return true;
            uniqueElements.add(nums[i]);
        }
        return false;
    }
}

File: my-folder/0217-contains-duplicate/solution.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        std::set<int> numbers;
        for (int num : nums) {
            if (numbers.contains(num)) {
                return true;
            }
            numbers.insert(num);
        }
        return false;
    }
};
